docs/version_bump_template.md

```diff
@@ -20,26 +20,29 @@ General components:
 
 - Rke2        
 ```
-1- flannel
-2- calico
-3- ingressController
-4- coredns
-5- metricsServer
-6- etcd
-7- containerd
-8- runc
+1- coredns
+2- metrics server
+3- etcd
+4- containerd
+5- runc
+6- crictl
+7- canalFlannel
+8- calico
+9- ingressController
+
 ```
 - k3s
 ```
 1- coredns
 2- metrics server
 3- etcd
 4- cni plugins
-5- traefik
-6- local-path-provisioner
-7- containerd
-8- klipper
-9- runc
+5- containerd
+6- runc
+7- crictl
+8- traefik
+9- local path provisioner
+10- klipper LB
```

Comment:
Let's add a reminder in this doc to update k3scomponentsCmdsLength and rke2componentsCmdsLength when we add new component in future https://github.com/rancher/distros-test-framework/pull/161/files#diff-e6edfe9d7866ecfc9abc604021947e006dd2dd7f831e434afe4dbecd4534d8ebR143

Reply:
done!